keycloak/keycloak-operator
# [DO NOT MERGE] Update the operator-sdk version from v0.10.0 to v0.11.0

## JIRA ID
https://issues.jboss.org/browse/INTLY-3929

## Additional Information
Update the operator-sdk version to v0.11.0 following this guide - https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#v011x

The newest version includes the controller-runtime version v0.2.0 which includes webhook refactor.

## Verification Steps
1. Run the operator on a cluster
2. Create a Keycloak and KeycloakRealm CR
3. Verify everything is working as expected

## Checklist:
